model/reminder.js

```diff
@@ -0,0 +1,17 @@
+const { Enum } = require('enumify');
+
+class Status extends Enum {
+  toString() {
+    return this.name.toLowerCase();
```

Comment:
I used lower case in order to not break the database schema and what returns the REST API, as well.